File: Mobile/app/src/main/java/Ticket.java
package com.example.mobile;

import com.google.gson.annotations.SerializedName;
import java.io.Serializable;

public class Ticket implements Serializable {

    // A anotação agora procura pelo campo "id", que é o que a API envia.
    @SerializedName("id")
    private int chamadoId;

    @SerializedName("titulo")
    private String titulo;

    @SerializedName("descricao")
    private String descricao;

    @SerializedName("status")
    private String status;

    @SerializedName("profissionalDesignado")
    private String profissionalDesignado;


    @SerializedName("solucao")
    private String solucao;



    public int getChamadoId() { return chamadoId; }
    public String getTitulo() { return titulo; }
    public String getDescription() { return descricao; }
    public String getStatus() { return status; }
    public String getProfessionalName() { return profissionalDesignado; }
    public String getSolucao() { return solucao; }
}

File: DesktopWPF - Bk/DesktopWPF/Models/Ticket.cs
using System;

namespace DesktopWPF.Models
{
    public class Ticket
    {
        public int Id { get; set; }
        public string Titulo { get; set; }
        public string Descricao { get; set; }
        public DateTime DataAbertura { get; set; }
        public string Status { get; set; }
        public int UsuarioId { get; set; }
        public string? ProfissionalDesignado { get; set; }
        public string? Solucao { get; set; }
    }
}